src/remotepublish.cmd.js

```diff
@@ -475,8 +476,15 @@ ${e}`);
       this.showNextStep(this._dryRun);
     } catch (e) {
       const message = 'Error while running the Publish command';
-      this.log.error(`${message}: ${e.stack}`, e);
-      throw new Error(message, e);
+      if (e.statusCode === 403) {
+        throw new Error(`${message}:
+The provided GITHUB_TOKEN is not authorized to act on behalf 
```

Comment:
@MarquiseRosier: is the token really the actor here? I think this should probably be rephrased to sth like _The provided GITHUB_TOKEN is not authorized to be used by the Helix Bot to update the purge config._

Reply:
the wording is technically correct:
see https://developer.github.com/apps/building-github-apps/authenticating-with-github-apps/#authenticating-as-an-installation

Reply:
Maybe I'm missing something, but I still think the actor in the sentence should be the bot, not the token which allows the bot to act... anyway, I forgot to add the _nitpicking_ label ;)

Reply:
Ahh snap that's a good point haha. I'll definitely submit a PR for another issue; and lump the rename in there! I'll get a review from you :D :)